integration/users/connection_test.go

```diff
@@ -94,9 +118,8 @@ func TestAuthentication(t *testing.T) {
 			password:            "password",
 			mechanisms:          []string{"SCRAM-SHA-256"},
 			connectionMechanism: "SCRAM-SHA-1",
-			errorMessage:        "Unable to use SCRAM-SHA-1 based authentication for user without any SCRAM-SHA-1 credentials registered",
 			topologyError:       true,
-			failsForFerretDB:    true,
+			errorMessage:        "Unable to use SCRAM-SHA-1 based authentication for user without any SCRAM-SHA-1 credentials registered",
```

Comment:
Could we assert exact error command/write error? That way, it can also check error code is the same.

Reply:
I see this is topology.Connection error that wrapped authentication error, so it won't be possible :see_no_evil: